status/k8s.go

```diff
@@ -17,6 +18,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/cilium/cilium-cli/defaults"
+	"github.com/cilium/workerpool"
+)
+
+const (
+	// DefaultWorkerCount is the numbner of the max workers used to gather the status
+	DefaultWorkerCount int = 64
```

Comment:
I'd suggest using a more conservative default (something like 5) knowing that users may manually override this number if need be.